.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
This file is not necessary. Add the `.vscode` to .gitignore so files like this will be ignored.